dnase/trackhub/update_tracks.bash

```diff
@@ -211,16 +211,27 @@ cd ${src}
 
 cp -R assets/${hub_type}/. ${hub_target}
 
-for i in "${genome_array[@]}"; do
-    [ "${i}" = "${customGenomeAssembly}" ] && continue
-    echo "genome ${i}" >> "${hub_target}/genomes.txt"
-    echo "trackDb ${i}/trackDb_001.txt" >> "${hub_target}/genomes.txt"
+# Add two-line stanzas for genomes native to the UCSC browser
+for cur_genome in "${genome_array[@]}"; do
+    # The custom assemblies have more complicated, non-standard stanzas, which are stored
+    # in the stub genomes.txt file in the assets/CEGS subdirectory, so we skip building them in this loop.
+    [ "${cur_genome}" = "${customGenomeAssembly}" ] || [ "${cur_genome}" = "t2t" ] && continue
+
+    echo "genome ${cur_genome}" >> "${hub_target}/genomes.txt"
+    echo "trackDb ${cur_genome}/trackDb_001.txt" >> "${hub_target}/genomes.txt"
     echo " " >> "${hub_target}/genomes.txt"
 done
 
 if [ -f "${assemblyBaseDir}/sequences/${customGenomeAssembly}/${customGenomeAssembly}.2bit" ]; then
     cp ${assemblyBaseDir}/sequences/${customGenomeAssembly}/${customGenomeAssembly}.2bit ${hub_target}/${customGenomeAssembly}/data/${customGenomeAssembly}.2bit
 fi
+
+if [[ "${hub_type}" == "CEGS" ]]; then
+    if [ ! -d "${hub_target}/t2t/data" ]; then
```

Comment:
one last question, how could this directory have been created before?

Reply:
Was just checking.  It is not though.  I'll take the if out.